configure.ac

```diff
@@ -203,7 +203,7 @@ AC_SUBST([FREEZE_MODULE_DEPS])
 AC_SUBST([PYTHON_FOR_BUILD_DEPS])
 
 AC_CHECK_PROGS([PYTHON_FOR_REGEN],
-  [python$PACKAGE_VERSION python3.10 python3.9 python3.8 python3.7 python3.6 python3 python],
+  [python$PACKAGE_VERSION python3.11 python3.10 python3.9 python3.8 python3.7 python3.6 python3 python],
```

Comment:
Why are we allowing `python3.8` and lower in the first place if we know that 3.9+ is required?

Reply:
If 3.9 or higher is required, we should remove the earlier and incompatible versions.

Reply:
Guido said he wanted to remove the 3.9+ dependency that crept into one of the scripts. Increasing the minimum required version might need more discussion as it could make bootstrapping harder for some.

Reply:
@erlend-aasland any objections to landing this?

Reply:
Nah, good to good for my part :)